store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStoreTest.java

```diff
@@ -580,4 +582,48 @@ void shouldDoNothingWhenCannotClose() throws Exception {
         }
      }
 
+     @Test
+     void shouldStoreTableCreatedTimeProperty() throws Exception {
+         // Given
+         final AccumuloProperties properties = PROPERTIES.clone();
+         String graphId = "graphId";
+
+         final AccumuloStore store = new MiniAccumuloStore();
+         // When
+
+         store.initialise(graphId, SCHEMA, properties);
+         LocalDateTime dateTime = LocalDateTime.now();
+         LocalDateTime storeDateTime = LocalDateTime.parse(store.getCreatedTime());
+         // Then
+         assertThat(storeDateTime).isBeforeOrEqualTo(dateTime);
+     }
+     @Test
+     void shouldThrowGafferRuntimeExceptionForGetCreatedTimeIfTableNotFound() throws Exception {
+        // Given
+        final AccumuloStore store = new MiniAccumuloStore();
+        store.initialise("graphId", SCHEMA, PROPERTIES);
+        GafferRuntimeException expectedException = new GafferRuntimeException("Error getting timestamp.");
+
+        // When
+        store.getConnection().tableOperations().delete("graphId");
+        GafferRuntimeException exception = assertThrows(GafferRuntimeException.class, () -> store.getCreatedTime());
+
+        // Then
+        assertThat(exception.getMessage()).isEqualTo(expectedException.getMessage());
```

Comment:
you could simplify this by removing lines 605 and 609 and changing the test to simply be 
`assertThatExceptionOfType(GafferRuntimeException.class).isThrownBy(() -> store.getCreatedTime()).withMessage("Error getting timestamp"); `

This could then be repeated on the test below :)